utils/payment-term.js

```diff
@@ -54,6 +55,18 @@ class PaymentTerm {
 		return checked.getAttribute('value');
 	}
 
+	getBaseAmount () {
+		const checkedDiv = this.$paymentTerm.querySelector('input:checked').parentElement.parentElement;
+		if (!checkedDiv) {
+			throw new Error('No payment term has been selected');
+		}
+		const baseElement = checkedDiv.getElementsByClassName(BASE_AMOUNT_CLASS)[0];
+		if (!baseElement) {
+			throw new Error('No base amount element was found');
+		}
+		return baseElement.getAttribute('value');
+	}
+
```

Comment:
if we use data-attribute, this simply becomes (dataset is a special key to get the data attributes of an element):
```suggestion
	getBaseAmount () {
		const checked = this.$paymentTerm.querySelector('input:checked');
		if (!checked) {
			throw new Error('No payment term has been selected');
		}
		const baseAmount = checked.dataset.baseAmount;

		return baseAmount;
	}

```

Reply:
yes, I prefer that solution